modules/battery_manager/README.md

```diff
@@ -0,0 +1,147 @@
+   of the battery with no visit labels (`Visit_label=NULL`) will be ignored.*
+ - FirstVisit:
+   - If `firstVisit` is set to `Y`, the test battery instance is applied
+   only if it is the first visit.
+   - If `firstVisit` is to `N`, the test battery instance is applied only if it
+   *not* the first visit.
+   - If `firstVisit` is set to null, the test battery instance is applied to any
+   visit.
+   - **NOTE:** *The `firstVisit` flag can allow to bypass all other rules in
+   some instances. In these instances, the `stage`, `CenterID`, `Visit_label`,
+   `AgeMinDays` and `AgeMaxDays` will not affect in any way the administration
+   of the instrument. Only the `subprojectID` value will impact if the
+   instrument gets administered or not; the `subprojectID` value must match
+   the timepoint's in order for the instrument to be administered in these
+   instances.*
+
+## Purpose
+The Battery Manager module allows users to **browse**, **add**,
+**edit**, **activate** and **deactivate** entries in the Test Battery. The
+Test Battery is used to determine which Instruments are administered at
+different timepoints.
+
+## Intended Users
+The Battery Manager module is used by study
+administrators.
+
+## Scope
+The Battery Manager module provides a tool for browsing, adding,
+editing, activating, and deactivating entries in the the Test Battery.
+
+### Browse
+The **browse** functionality is under the `Browse` tab. Users
+can browse entries in the Test Battery and filter the entries using the
+`Selection Filters`.
+
+### Add
+The **add** functionality is under the `Add` tab. Users can add
+an entry by filling out the required fields. If the entry already exists
+in the Test Battery table, the entry cannot be added.
+
+### Edit
+The **edit** functionality can be accessed in the Menu Table
+under the `Browse` tab. Users can navigate to an entry's edit window
+by clicking on the corresponding `Edit` link in the `Edit Metadata`
+column. The edit window will have a form that is populated with the
+values of that entry. Editing entails deactivating the original entry
+and creating a new entry with the edited values.  The following rules
+apply when editing:
+- Users cannot edit an entry if they have made no
+changes in the form.
+- Users cannot edit an entry if it becomes the
+same as another active entry in the Test Battery.
+- If the edited entry becomes the same as another deactivated entry in the
+Test Battery, users will be given the option to activate the other entry and
+deactivate the original one.
+
+### Activate
+The **activate** functionality can be accessed in the
+`Browse` tab, `Edit` window, and the `Add` tab.
+
+#### Browse tab
+In the `Change Status` column of the Menu Table, users
+can directly activate an entry by pressing the **Activate** button.
+
+#### Interactions with LORIS
+Changes, additions and deletion of data in this module affects the test
+battery assigned to a candidate at each timepoint
+
+#### Edit window
```

Comment:
The descriptions of what happens on each tab/page aren't interactions, they belong in the frontend user help section, not the README.

Reply:
@driusan Could you clarify which of these headings and sections should be removed?